sdk/python/kfp/dsl/_container_op.py

```diff
@@ -268,5 +269,38 @@ def add_node_selector_constraint(self, label_name, value):
     self.node_selector[label_name] = value
     return self
 
+  def add_pod_annotation(self, name: str, value: str):
+    """Adds a pod's metadata annotation.
+
+    Args:
+      name: The name of the annotation.
+      value: The value of the annotation.
+    """
+
+    self.pod_annotations[name] = value
+    return self
+
+
+  def set_tpu(self, tpu_cores: str, tpu_resource: str, tf_version: str):
```

Comment:
tpu_cores: int

Does it make sense to provide defaults?

Reply:
I don't think there is a good value for default. User should decide how many TPU they want before enable it.

Reply:
I worded my comment wrong. I meant the defaults for `tpu_resource` and `tf_version`.